extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -53,6 +53,81 @@ tags: {  # <amp-story>
       allowed_protocol: "https"
     }
   }
+  attrs: {
+    name: "title"
+    mandatory: true
+  }
+  attrs: {
+    name: "publisher"
+    mandatory: true
+  }
+  attrs: {
+    name: "publisher-logo-src"
+    mandatory: true
+    value_url: {
+      allowed_protocol: "http"
+      allowed_protocol: "https"
+    }
+  }
+  attrs: {
+    name: "poster-portrait-src"
+    mandatory: true
+    value_url: {
+      allowed_protocol: "http"
+      allowed_protocol: "https"
+    }
+  }
+  attrs: {
+    name: "poster-square-src"
+    value_url: {
+      allowed_protocol: "http"
+      allowed_protocol: "https"
+    }
+  }
+  attrs: {
+    name: "poster-landscape-src"
+    value_url: {
+      allowed_protocol: "http"
+      allowed_protocol: "https"
+    }
```

Comment:
@honeybadgerdontcare Should we set these new URL attributes to `allow_relative: false` to avoid need for rewriting?

Reply:
@choumx yes, lets do that since these are new.

Reply:
Done.